### ISSUE TITLE
5Ghz Client / WDS

### ISSUE THREAD

**Issue by gustavo-momente** [zaccareal/openwrt-archer-c2#2]
Would be possible to compile 5Ghz drivers with Client mode and WDS enabled?

**Comment by zaccareal** [comment]
Of course, in menuconfig mediatek > drivers > mt7610e and set what you need.
Later you have to configure via console because for now it's not possible through LuCi

**Comment by gustavo-momente** [comment]
Thanks a lot! But right now I don't have a machine setup in a way that I could compile the project. Would you be so kinda as to provide compiled binaries with those options enabled?

**Comment by gustavo-momente** [comment]
@zaccareal after finally setting up a build env, I could follow your instructions. But after flashing the resulting image I can't connect to any 5Ghz network using the modem.

Did you ever test the 5Ghz network on client mode?

**Comment by zaccareal** [comment]
Me too I struggled with client mode but finally i got it works.
After apclii0 interface configuration remember to set rai0/(apclii0) Channel according to your 5ghz network.
`iwconfig apclii0` command and System log/dmesg show connection status/info

**Comment by gustavo-momente** [comment]
@zaccareal could you share how did you manage to configure apclii0? I didn't even know I had to edit and create that interface.

**Comment by zaccareal** [comment]
To configure STA mode follow these steps: https://forum.openwrt.org/viewtopic.php?pid=358223#p358223 (you don't need to load driver)
Remember to set appropriate Channel with `iwpriv rai0 set Channel=36`

Here you can find detailed information about configuration:
http://sourceforge.net/p/wive-ng/wive-ng-mt/ci/05d0fa0ec200e4201f3fca74694ed2dee6de6aff/tree/docs/DriversNotes/2011_0124_RT3x5x_5350_Linux_SoftAP_v2.5.0.0_ReleaseNotes.pdf?format=raw

**Comment by gustavo-momente** [comment]
Thanks I lot! I followed your steps and I did manage to connect as a Client! Thanks a lot for the help.
Do you think in the future it would be possible to do this configuration by setting the interface to `'sta' mode` in `/etc/config/wireless`? 

Also, have you ever tried to make a relayd bridge? If you did, would you be so kind to share your configuration?

**Comment by zaccareal** [comment]
Soon i'll add STA mode LuCi support, the code is ready, now i have not access to the router to test it
I never tested relayd bridge

**Comment by gustavo-momente** [comment]
Oh! That's awesome. If you want I can beta test that for you.

2017-10-04 10:57 GMT-03:00 zaccareal <notifications@github.com>:
